src/main/java/gui/utility/NavigationHelper.java

```diff
@@ -117,6 +119,9 @@ public static void enterPomodoroPage(MouseEvent event) throws IOException {
      * @throws IOException if the resource file cannot be found
      */
     public static void enterSettingsPage(MouseEvent event) throws IOException {
+        FXMLLoader loader = new FXMLLoader();
+        loader.setLocation(Objects.requireNonNull(NavigationHelper.class.getResource("/setting.fxml")));
```

Comment:
can you include files related to the settings page here too?

Reply:
I asked @halfxblood  to open a new PR for setting.
I think he is working on that branch